### PR TITLE
Add documentation for renault.ac set schedules

### PR DESCRIPTION
**Pull request by vhkristof**
## Proposed change

This PR adds documentation for the new service that is implemented by https://github.com/home-assistant/core/pull/125006

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/125006
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a new action to update the air conditioning schedule for vehicles, enhancing user control over vehicle climate settings.
- **Improvements**
	- Updated the existing action for charging schedules to align with the new structure, providing clearer guidelines for users.
- **Documentation**
	- Enhanced documentation with detailed notes and examples for both air conditioning and charging schedule actions, improving user understanding and usability.

### CODE REVIEW

**Comment by vhkristof** [comment]
Did a fuck-up in my previous PR: #34521

**Comment by vhkristof** [comment]
@epenet  Could you review this PR ?

File: source/_integrations/renault.markdown [hunk 1]
```diff
@@ -58,6 +58,37 @@ Cancel A/C on vehicle.
...
+Update AC schedule on vehicle.
+
+  | Data attribute | Required | Description | Example |
+  | ---------------------- | -------- | ----------- | ------- |
+  | `vehicle`| yes | device_id of the vehicle | |
+  | `schedules` | yes | Schedule details. Can be a single schedule or a list of schedules | see [example below](#ac_schedule_example) |
+
+Notes:
+
+- `schedules` can contain one or more schedules which are set within the same call
```
Comment (c0ffeeca7) [inline source/_integrations/renault.markdown]:
```suggestion
- `schedules` can contain one or more schedules which are set within the same call.
```

File: source/_integrations/renault.markdown [hunk 2]
```diff
@@ -58,6 +58,37 @@ Cancel A/C on vehicle.
...
+- The `id` is compulsory on each `schedule` (should be 1 to 5 depending on the vehicle).
```
Comment (c0ffeeca7) [inline source/_integrations/renault.markdown]:
```suggestion
- The `id` is compulsory on each `schedule` (should be 1 to 5, depending on the vehicle).
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @vhkristof :+1:

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**